packages/svelte/package.json

```diff
@@ -1,57 +1,63 @@
 {
-	"name": "@aws-amplify/ui-svelte",
-	"version": "0.0.1",
-	"type": "module",
-	"main": "dist/index.js",
-	"svelte": "index.js",
-	"exports": {
-		".": {
-			"import": "./dist/index.js"
-		},
-		"./styles.css": "./dist/styles.css"
-	},
-	"browser": {
-		"./styles.css": "./dist/styles.css"
-	},
-	"scripts": {
-		"dev": "svelte-kit dev",
-		"build": "svelte-kit build",
-		"postbuild": "svelte-kit package",
-		"package": "svelte-kit package",
-		"preview": "svelte-kit preview",
-		"check": "svelte-check --tsconfig ./tsconfig.json",
-		"check:watch": "svelte-check --tsconfig ./tsconfig.json --watch",
-		"lint": "prettier --ignore-path .gitignore --check --plugin-search-dir=. . && eslint --ignore-path .gitignore .",
-		"format": "prettier --ignore-path .gitignore --write --plugin-search-dir=. ."
-	},
-	"devDependencies": {
-		"@sveltejs/adapter-auto": "next",
-		"@sveltejs/adapter-node": "^1.0.0-next.56",
-		"@sveltejs/adapter-static": "^1.0.0-next.23",
-		"@sveltejs/kit": "next",
-		"@typescript-eslint/eslint-plugin": "^4.31.1",
-		"@typescript-eslint/parser": "^4.31.1",
-		"eslint": "^7.32.0",
-		"eslint-config-prettier": "^8.3.0",
-		"eslint-plugin-svelte3": "^3.2.1",
-		"prettier": "^2.4.1",
-		"prettier-plugin-svelte": "^2.4.0",
-		"svelte": "^3.44.0",
-		"svelte-check": "^2.2.6",
-		"svelte-preprocess": "^4.9.4",
-		"svelte2tsx": "^0.4.12",
-		"tslib": "^2.3.1",
-		"typescript": "^4.4.3"
-	},
-	"peerDependencies": {
-		"aws-amplify": "^4.2.2"
-	},
-	"dependencies": {
-		"@aws-amplify/ui": "^3.0.5",
-		"@rollup/plugin-replace": "^3.0.0",
-		"nanoid": "^3.1.30",
-		"qrcode": "^1.5.0",
-		"svelte2tsx": "^0.4.12",
-		"xstate": "^4.26.1"
-	}
+  "name": "@aws-amplify/ui-svelte",
+  "version": "0.0.1",
+  "type": "module",
+  "main": "dist/index.js",
+  "svelte": "dist/index.js",
+  "exports": {
+    "./package.json": "./package.json",
+    ".": {
+      "import": "./dist/index.js"
+    },
+    "./styles.css": "./dist/styles.css"
+  },
+  "browser": {
+    "./styles.css": "./dist/styles.css"
+  },
+  "files": [
+    "dist",
+    "LICENSE"
+  ],
+  "scripts": {
+    "dev": "svelte-kit dev",
+    "build": "svelte-kit build",
+    "postbuild": "svelte-kit package && shx cp ../ui/dist/styles.css dist/styles.css",
```

Comment:
For now I'm just using a library called shx (cross platform) to copy the styles to the dist folder. Later, I'll see if it's possible with a vite/rollup plugin. For some reason SvelteKit won't recognize the styles import otherwise, from `ui`.